packages/next-swc/crates/next-api/src/app.rs

```diff
@@ -578,7 +578,7 @@ impl AppEndpoint {
         };
         let build_manifest_output = Vc::upcast(VirtualOutputAsset::new(
             node_root.join(format!(
-                "app{original_name}/build-manifest.json",
+                "server/app{original_name}/build-manifest.json",
```

Comment:
I didn't put this in `.next/server` because by default it's at the root of `.next`. Some manifests are in `.next` directly, some are in `.next/server`. This only contains client-side things.